contracts/lib/BasicOrderFulfiller.sol

```diff
@@ -83,7 +83,7 @@ contract BasicOrderFulfiller is OrderValidator {
             orderType := and(calldataload(BasicOrder_basicOrderType_cdPtr), 3)
 
             // Divide basicOrderType by four to derive the route.
-            route := div(calldataload(BasicOrder_basicOrderType_cdPtr), 4)
+            route := shr(2, calldataload(BasicOrder_basicOrderType_cdPtr))
```

Comment:
99% sure the yul optimizer can do this on its own?

Reply:
may as well be explicit about it 🤷

Reply:
> 99% sure the yul optimizer can do this on its own?

Saved gas in my test at least